index.html

```diff
@@ -0,0 +1,2059 @@
+                    BY<span class="span-debate font-weight-bold">
+                      MAJOR NEILL FRANKLIN (RET.)</span
+                    >
+                  </p>
+                </div>
+              </div>
+              <div class="vs-container">
+                <div class="line"></div>
+                <div class="vs">VS</div>
+              </div>
+              <div class="media">
+                <div class="media-body">
+                  <h5 class="mt-0 mb-1 font-weight-bold">
+                    Legalizing Pot Is a Catastrophic Mistake
+                  </h5>
+                  <p>
+                    BY
+                    <span class="span-debate font-weight-bold"
+                      >KEVIN SABET</span
+                    >
+                  </p>
+                </div>
+                <img
+                  class="d-flex ml-3"
+                  src="https://d.newsweek.com/en/full/111059/kevin-sabet.png?w=150&h=150&f=2d885279d159e97deb11590f95b67c10"
+                  alt="Kevin Sabet"
+                  style="width: 64px; height: 64px;"
+                />
+              </div>
+              <div class="line"></div>
+            </div>
+            <div class="opinion gap">
+              <div class="h2-div font-weight-bold py-4">OPINION</div>
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Slam"
+                  src="https://d.newsweek.com/en/full/1626435/salam-al-marayati.png?w=63&h=63&f=6b7145fbfece475556bde636cd82c729"
+                  style="width: 64px; height: 64px;"
+                />
+                <div class="media-body">
+                  <h5 class="mt-0 font-weight-bold">
+                    We Need More American Muslims in Public Life
+                  </h5>
+                  <p class="font-weight-bold">BY SALAM AL-MARAYATI</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Dyllón"
+                  src="https://d.newsweek.com/en/full/1626108/dyllon-burnside.png?w=63&h=63&f=5cf9e387cc40a173b33ead8bbaf5558b"
+                  style="width: 64px; height: 64px;"
+                />
+                <div class="media-body">
+                  <h5 class="mt-0 font-weight-bold">
+                    The HIV Pandemic Is Still Raging—and Won't Stop Until We End
+                    the Stigma
+                  </h5>
+                  <p class="font-weight-bold">BY DYLLÓN BURNSIDE</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Mohammed"
+                  src="https://d.newsweek.com/en/full/1602983/mohammed-alsherebi.png?w=63&h=63&f=f3cb5702ecaac97c55ce6e0f1174eecb"
+                  style="width: 64px; height: 64px;"
```

Comment:
- [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)